pybamm/models/submodels/interface/base_interface.py

```diff
@@ -141,15 +141,17 @@ def _get_exchange_current_density(self, variables):
 
         return j0
 
-    def _get_open_circuit_potential(self, variables):
+    def _get_open_circuit_potential(self, variables, phase=None):
         """
         A private function to obtain the open circuit potential and entropic change
 
         Parameters
         ----------
         variables: dict
             The variables in the full model.
-
+        phase : string
```

Comment:
Using `1` or `2` doesn't give much information. Could you use more informative names ('graphite' and 'silicon')?

Reply:
They are general names, not only for graphite and silicon